front/components/vaults/VaultResourcesList.tsx

```diff
@@ -210,10 +268,22 @@ export const VaultResourcesList = ({
           </DropdownMenu>
         )}
         {category === "files" && (
-          <Button label="Add folder" onClick={() => {}} icon={PlusIcon} />
+          <Button
```

Comment:
`nit` I'd have put the button in the modal itself so the logic is tight together and we don't need to handle many state in this list component.

Reply:
will do in a next PR since I've done it for managed datasources